.spelling

```diff
@@ -11,6 +11,7 @@ adb
 add-on
 addon
 addons
+addons_daily
```

Comment:
drive-by: technical terms or code type things should be surrounded by backticks in the main text, which will make them pass the spell checker - `addons_daily`, `main_summary` and `v1` fall into this category.

Reply:
Thanks for the heads up!